src/commands/hexpire.json

```
@@ -0,0 +1,118 @@
+{
+    "HEXPIRE": {
+        "summary": "Set expiry time on hash fields.",
+        "complexity": "O(1) for each field assigned with TTL, so O(N) to add N TTLs when the command is called with multiple fields.",
+        "group": "hash",
+        "since": "9.0.0",
+        "arity": -6,
+        "function": "hexpireCommand",
+        "command_flags": [
+            "WRITE",
+            "FAST"
+        ],
+        "acl_categories": [
+            "HASH"
+        ],
+        "key_specs": [
+            {
+                "flags": [
+                    "RW",
+                    "UPDATE"
+                ],
+                "begin_search": {
+                    "index": {
+                        "pos": 1
+                    }
+                },
+                "find_keys": {
+                    "range": {
+                        "lastkey": 0,
+                        "step": 1,
+                        "limit": 0
+                    }
+                }
+            }
+        ],
+        "reply_schema": {
+            "description": "List of values associated with the result of setting expiry on the specific fields, in the same order as they are requested.",
```

nit - be explicit about the return type in the description? 

```suggestion
            "description": "List of integer codes indicating the result of setting expiry on each specified field, in the same order as the fields are requested.",
```